SPECS/python-more-itertools/python-more-itertools.signatures.json

```diff
@@ -1,6 +1,5 @@
-{
-    "Signatures": {
-     "more-itertools-8.13.0.tar.gz": "a42901a0a5b169d925f6f217cd5a190e32ef54360905b9c39ee7db5313bfec0f"
-    }
-   }
-   
+{
+  "Signatures": {
+    "more-itertools-10.2.0.tar.gz": "8fccb480c43d3e99a00087634c06dd02b0d50fbf088b380de5a41a015ec239e1"
+  }
+}
```

Comment:
Nit: There was a recent [PR](https://github.com/microsoft/CBL-Mariner/pull/7147 ) to ensure there is a newline at the end of every signature file